src/vma/sock/sock-redirect.cpp

```diff
@@ -613,6 +613,9 @@ int vma_dump_fd_stats(int fd, int log_level)
 	return -1;
 }
 
+/* Multi Packet Receive Queue functionality is deprecated
+ * and not going to be supported in the future release
```

Comment:
\* and **is** not going to be supported in future release**s**

Reply:
updated